src/strongbox/main.clj

```diff
@@ -60,6 +66,7 @@
     (gui/start))
 
   (watch-for-gui-restart)
+  (shutdown-hook)
```

Comment:
perhaps move out of `start` ? what happens if we call `restart`? multiple shutdown hooks??

Reply:
yup:

```
strongbox.main=> (System/exit 0)
2020-06-22 13:59:24.205 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.205 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.206 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.206 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.206 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.206 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.206 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.207 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.207 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.207 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.207 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.207 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.207 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.208 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.208 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.208 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.210 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.209 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.210 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.210 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.211 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.211 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.211 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.212 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.212 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.212 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.211 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.213 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.213 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.213 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.213 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.213 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.213 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.213 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.213 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.214 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.214 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.214 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.214 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.214 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.215 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.215 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.215 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.215 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.215 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.216 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.216 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.216 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.215 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.216 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.215 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.216 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.216 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.216 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.217 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.217 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.217 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.217 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.217 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.217 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.217 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.218 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.218 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.218 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.218 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.219 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.219 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.219 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.219 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.219 INFO [strongbox.ui.gui:980] - stopping gui
2020-06-22 13:59:24.219 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.220 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.220 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.220 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.219 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.220 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.221 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.221 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.221 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.222 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.223 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.224 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.224 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.225 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.227 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.228 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.229 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.230 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.230 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.231 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.231 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.232 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.232 INFO [strongbox.core:1312] - stopping app
2020-06-22 13:59:24.233 INFO [strongbox.core:1312] - stopping app

```